salt/utils/gitfs.py

```diff
@@ -486,11 +486,15 @@ def _val_cb(x, y):
             ).replace(
                 "/", "_"
             )  # replace "/" with "_" to not cause trouble with file system
+
         self._cache_hash = salt.utils.path.join(cache_root, self._cache_basehash)
         self._cache_basename = "_"
         if self.id.startswith("__env__"):
             try:
-                self._cache_basename = self.get_checkout_target()
+                self._cache_basename = self.get_checkout_target().replace(
+                    "/", "-"
+                )  # replace '/' with '-' to not cause trouble with file-system
```

Comment:
We're using `_` above, why switch to `-` here?

Reply:
This line used https://github.com/saltstack/salt/blob/2b44a6ce9a9c387b08ff3abb2bb025c5acece112/salt/utils/gitfs.py#L3387 with gitfs, hence thought I'd do same, but could be underscore like https://github.com/saltstack/salt/blob/2b44a6ce9a9c387b08ff3abb2bb025c5acece112/salt/utils/gitfs.py#L483-L488 if you prefer